README.md

```diff
@@ -0,0 +1,86 @@
+# Ansys actions
+
+[![ansys][Ansys badge]][`actions.docs.ansys.com`]
```

Comment:
Thanks for fixing this, @SerTetora. We are not using the inline code highlights for links. Thus, let's remove the back-ticks in the links.

```suggestion
[![ansys][Ansys badge]][actions.docs.ansys.com]
```